values.yaml

```diff
@@ -962,6 +967,22 @@ jobs:
     #     - run
     #     - --tail
     #     - custom_pack.warn_about_upgrade
+
+# st2canary has settings used for temporary chart hook resources
+# like a pod that validates packs.volumes before installs/upgrades.
```

Comment:
Just a bit more info about the `st2canary` would be good here and mentioning a fact the pod will be deleted if the configuration is good.